fcp-0010.md

```diff
@@ -0,0 +1,44 @@
+---
+authors: Warner Losh <imp@bsdimp.com>
+state: draft
+---
+
+# FCP 10: PC Card Deprecation Notice
+
+PC Card has not been relevant in years. It's inclusion in the tree is holding us back.
+
+## Problem Statement
+
+PC Card used to be very relevant. However, the only reason we have a
+number of really old network drivers is due to PC Card. If we were to
+remove it, many of them could be deleted. PC Card is a 16-bit only
+standard. Most relevant hardware that FreeBSD runs on has CardBus
+slots, and the 32-bit PCI cards are still well supported.
+
+There's little enough traffic on CardBus cards in the mobile mailing
+list, and none for PC Cards in many years.
+
+## Proposed Solution
+
+We will remove PC Card from the tree. We will also remove drivers that are
+only in the tree for PC Card support, and all the PC Card attachments.
+
+The unforunately named /etc/pccard_ether, run when a network card is
```

Comment:
We should probably rename it.

Reply:
We can't. Or at least it would require several steps.  It requires a new devd.conf setup which has been dodgy to require in the past as it is often configured by users... it hasn't seemed worth the effort as its name has become less relevant over time. It's also orthogonal to the removal.